docker-image-src/common/neo4jlabs-plugins.json

```diff
@@ -1,13 +1,12 @@
 {
   "apoc": {
-    "versions": "https://github.com/neo4j/apoc/blob/master/versions.json",
+    "versions": "https://neo4j-contrib.github.io/neo4j-apoc-procedures/versions.json",
     "properties": {
       "dbms.security.procedures.unrestricted": "apoc.*"
     }
```

Comment:
This apoc key needs to fetch from neo4j-contrib because it's used by all versions of pre-5.0 Neo4j. All of those versions of Neo4j use neo4j-contrib/apoc and need to fetch the neo4j-contrib versions.json.